server/services/store/storetests/initialize.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package storetests
+
+import (
+	"testing"
+	"github.com/mattermost/focalboard/server/services/store"
+	"github.com/stretchr/testify/require"
+)
+
+
+func StoreTestInitializeStore(t *testing.T, setup func(t *testing.T) (store.Store, func())) {
+	t.Run("Initialize Template", func(t *testing.T) {
+		store, tearDown := setup(t)
+		defer tearDown()
+		testInitializeTemplate(t, store)
+
+	})
+
+}
+
+func testInitializeTemplate(t *testing.T, store store.Store) {
+        t.Run("InitializeTemplate",func(t *testing.T) {
+
+	err := store.InitializeTemplates()
+	require.NoError(t, err)
```

Comment:
We should also validate that the templates were indeed inserted into the database.